deployments/charts/kuma/templates/cp-rbac.yaml

```diff
@@ -104,75 +103,63 @@ rules:
       - update
       - patch
       - delete
+  {{- if .Values.cni.enabled }}
   - apiGroups:
-      - ""
+      - k8s.cni.cncf.io
     resources:
-      - events
+      - network-attachment-definitions
     verbs:
-      - create
-      - patch
+      - get
+      - list
+      - watch
   - apiGroups:
-      - ""
+      - apiextensions.k8s.io
     resources:
-      - services
+      - customresourcedefinitions
     verbs:
       - get
-      - delete
       - list
       - watch
-      - create
+  - apiGroups: # required for taint controller
+      - ""
+    resources:
+      - nodes
+    verbs:
       - update
```

Comment:
I understand we can disable the taint controller by setting the right option in the control plane config. I'm wondering if we could expose that setting in `values.yaml`, so disabling it there would also make these permissions unnecessary. What do you think?

Reply:
We set it to true when cni is enabled https://github.com/kumahq/kuma/blob/master/deployments/charts/kuma/templates/_helpers.tpl#L310-L316

Reply:
What I meant is that it's possible to run the CNI without the taint controller by setting an env var like `KUMA_RUNTIME_KUBERNETES_NODE_TAINT_CONTROLLER_ENABLED` in the control plane runtime. Maybe we could expose this as a `values.yaml` option so when it's disabled, we also skip adding the related permission. It's not a blocker, just a suggestion to consider.